Makefile-libostree.am

```diff
@@ -226,7 +226,11 @@ libostree_1_la_CFLAGS += $(OT_DEP_CURL_CFLAGS)
 libostree_1_la_LIBADD += $(OT_DEP_CURL_LIBS)
 else
 if USE_LIBSOUP
+if USE_LIBSOUP3
+libostree_1_la_SOURCES += src/libostree/ostree-fetcher-soup3.c
+else
 libostree_1_la_SOURCES += src/libostree/ostree-fetcher-soup.c
+endif
```

Comment:
I think like in the fetcher code itself, libsoup (i.e. soup2) and libsoup3 should be totally separate if possible.

Reply:
I redid this part so they're a bit more separated at the expense of a little duplication.